controls/cis_rhel7.yml

```diff
@@ -1226,3 +1226,322 @@ controls:
     automated: no
     related_rules:
     - service_ip6tables_enabled
+
+  - id: 4.1.1.1
+    title: Ensure auditd is installed (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: partially # we do not check for audit-libs package
+    rules:
+    - package_audit_installed
+
+  - id: 4.1.1.2
+    title: Ensure auditd service is enabled and running (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - service_auditd_enabled
+
+  - id: 4.1.1.3
+    title: Ensure auditing for processes that start prior to auditd is enabled (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - grub2_audit_argument
+
+  - id: 4.1.2.1
+    title: Ensure audit log storage size is configured (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - auditd_data_retention_max_log_file
+    - var_auditd_max_log_file=6
+
+  - id: 4.1.2.2
+    title: Ensure audit logs are not automatically deleted (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - auditd_data_retention_max_log_file_action
+    - var_auditd_max_log_file_action=keep_logs
+
+  - id: 4.1.2.3
+    title: Ensure system is disabled when audit logs are full (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - auditd_data_retention_space_left_action
+    - var_auditd_space_left_action=email
+    - auditd_data_retention_action_mail_acct
+    - var_auditd_action_mail_acct=root
+    - auditd_data_retention_admin_space_left_action
+    - var_auditd_admin_space_left_action=halt
+
+  - id: 4.1.2.4
+    title: Ensure audit_backlog_limit is sufficient (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    notes: <-
+      Note that currently the value is hardcoded to 8192
+    rules:
+    - grub2_audit_backlog_limit_argument
+
+  - id: 4.1.3
+    title: Ensure events that modify date and time information are collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_time_adjtimex
+    - audit_rules_time_clock_settime
```

Comment:
Rule `audit_rules_time_clock_settime` seems to have an extra filter `-F a0=0x0`.